src/session.ts

```diff
@@ -336,7 +344,11 @@ export class Session extends ServiceObject {
    *   }
    * });
    */
-  keepAlive(callback?): void | Promise<void> {
+  keepAlive(): Promise<[r.Response]>;
+  keepAlive(callback: NormalCallback<r.Response>): void;
```

Comment:
Move these overloads on top of the JSDoc, otherwise this documentation will be removed by TypeScript 
Dave's [comment](https://github.com/googleapis/nodejs-spanner/pull/627#pullrequestreview-246182206), point two

Reply:
Thanks My bad